docs/src/pages/components/data-grid/selection/ControlledSelectionGrid.tsx

```diff
@@ -15,8 +15,8 @@ export default function ControlledSelectionGrid() {
     <div style={{ height: 400, width: '100%' }}>
       <DataGrid
         checkboxSelection
-        onSelectionModelChange={(newSelection) => {
-          setSelectionModel(newSelection.selectionModel);
+        onSelectionModelChange={(newSelectionModel) => {
+          setSelectionModel(newSelectionModel);
```

Comment:
I like this 👍

Reply:
I would have personally done
```suggestion
        onSelectionModelChange={(event, newSelectionModel, details) => {
          setSelectionModel(newSelectionModel);
```

but it's already one step in the right direction, IMHO